Formula/rdkit.rb

```diff
@@ -65,6 +66,10 @@ def install
     system "cmake", ".", *args
     system "make"
     system "make", "install"
+
+    site_packages = "lib/python#{py3ver}/site-packages"
+    pth_contents = "import site; site.addsitedir('#{libexec/site_packages}')\n"
+    (prefix/site_packages/"homebrew-rdkit.pth").write pth_contents
```

Comment:
```suggestion
    site_packages = "lib/python#{py3ver}/site-packages"
    (prefix/site_packages/"homebrew-rdkit.pth").write libexec/site_packages
```
I _think_ `pth` files can now just be a list of paths, instead of Python code that calls `addsitedir`.

Ref: https://docs.python.org/3/library/site.html

On a different note, maybe it's better if we can have `rdkit` just install the Python module into `lib/"python#{py3ver}/site-packages"` directly, in light of #73875.

Reply:
Thank you for your review. I have made the changes as you suggested.